HippoMocks/hippomocks.h

```diff
@@ -85,8 +85,15 @@ extern "C" __declspec(dllimport) void WINCALL DebugBreak();
 #define FUNCTION_STRIDE 1
 #endif
 
+#if defined (__SUNPRO_CC)
+#define SUNPRO_CC_VTABLE_INDEX 2
```

Comment:
What's in the first two then, on SUNPRO_CC? Is it the base offset and RTTI object pointer?